### PR TITLE
kube-proxy of rp nodes connecting to tp masters

### LINKED ISSUE #1346: [kube-up][scale-out] kube-proxy on RP nodes connect to RP master
**What happened**:
kube-proxy pods running on RP nodes are connecting to RP master (external IP address)

**What you expected to happen**:
in scale-out cluster, kube-proxy should connect to TP master servers

**How to reproduce it (as minimally and precisely as possible)**:
at poc-2022-01-30 branch, start 1 tp 1 rp 1 worker scale-out cluster by kube-up.sh

**Anything else we need to know?**:

**Environment**:
- Arktos version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools: kube-up scale-out 1+1x1
- Network plugin and version (if this is a network-related bug):
- Others:

### PR DESCRIPTION
**Pull request by h-w-chen**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
It sets --tenant-server-kubeconfig arg pointing to TP masters for kube-proxy at RP nodes.

**Which issue(s) this PR fixes**:

Fixes #1346 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cluster/gce/gci/configure-helper-common.sh [hunk 1]
```diff
@@ -1367,7 +1367,19 @@ function prepare-log-file {
...
-  local -r kubeconfig="--kubeconfig=/var/lib/kube-proxy/kubeconfig"
+  local kubeconfig="--kubeconfig=/var/lib/kube-proxy/kubeconfig"
+  if [[ "${ARKTOS_SCALEOUT_SERVER_TYPE:-}" == "node" ]]; then
+    echo "DBG:Set tenant-server-kubeconfig parameters:  ${TENANT_SERVER_KUBECONFIGS}"
+    kubeconfig+=" --tenant-server-kubeconfig=${TENANT_SERVER_KUBECONFIGS}"
+  fi
+
+  local proxy_tpconfig_mount=""
+  local proxy_tpconfig_volume=""
+  if [[ "${ARKTOS_SCALEOUT_SERVER_TYPE:-}" == "node" ]]; then
```
Comment (yb01) [inline cluster/gce/gci/configure-helper-common.sh:1378]: nit: the two if blocks can be combined.

**Review by yb01 — APPROVED** [review]
/lgtm

**Review by sonyafenge — APPROVED** [review]
/lgtm

**Pull request merged**